controllers/multicluster/clusterregistry/README.md

```diff
@@ -0,0 +1,92 @@
+# Multicluster Technical Documents
+
+This documents not only describe the behavior of `clusterregistry`, it
+also describes the technical framework which can help the chaos mesh developers
+to develop the multicluster application.
+
+A `clusterregistry` will manage all controllers running on a remote cluster. The
+construction of these controllers (managers) will be managed by `fx`. The main
```

Comment:
```suggestion
A `clusterregistry` will manage all controllers watching on a remote cluster. The
construction of these controllers (managers) will be managed by `fx`. The main
```

`running` makes confusion for me, because the logic would not "run" on a remote cluster.